src/Microsoft.Win32.Primitives/ref/Microsoft.Win32.Primitives.cs

```diff
@@ -8,13 +8,15 @@
 
 namespace System.ComponentModel
 {
-    public partial class Win32Exception : System.Exception
+    public partial class Win32Exception : System.Runtime.InteropServices.ExternalException
```

Comment:
Though, looking again, I see that for whatever reason this one does in desktop:
https://referencesource.microsoft.com/#System/compmod/system/componentmodel/Win32Exception.cs,29
It's not necessary, but I'll add it for easy-of-comparison.  The type implements the interface regardless.